File: monitor/v1/event/read-example-date-filter/read-example-date-filter.5.x.rb
# Download the helper library from https://www.twilio.com/docs/ruby/install
require 'rubygems'
require 'twilio-ruby'

# Your Account Sid and Auth Token from twilio.com/console
# DANGER! This is insecure. See http://twil.io/secure
account_sid = 'ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX'
auth_token = 'your_auth_token'
@client = Twilio::REST::Client.new(account_sid, auth_token)

events = @client.monitor.events.list(
                                  end_date: Date.new(2015, 4, 1),
                                  start_date: Date.new(2015, 3, 1)
                                )

events.each do |record|
  puts record.sid
end

File: monitor/v1/event/read-example-date-filter/read-example-date-filter.5.x.cs
// Install the C# / .NET helper library from twilio.com/docs/csharp/install

using System;
using Twilio;
using Twilio.Rest.Monitor.V1;


class Program 
{
    static void Main(string[] args)
    {
        // Find your Account Sid and Token at twilio.com/console
        const string accountSid = "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
        const string authToken = "your_auth_token";

        TwilioClient.Init(accountSid, authToken);

        var events = EventResource.Read(
            endDate: new DateTime(2015, 4, 1, 0, 0, 0),
            startDate: new DateTime(2015, 3, 1, 0, 0, 0)
        );

        foreach(var record in events)
        {
           Console.WriteLine(record.Sid);
        }
    }
}
